### PR TITLE
add new recipe for Pal2nal

### PR DESCRIPTION
**Pull request by liaochenlanruo**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by liaochenlanruo** [comment]
please help me merge the pull requests @epruesse, and I also want to join bioconda. Thanks!

File: recipes/pal2nal/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+about:
+  home: http://www.bork.embl.de/pal2nal/
+  license: GPL
+  license_family: GPL
+  license_file: README
+  summary: robust conversion of protein sequence alignments into the corresponding codon alignments
+
+extra:
+  identifiers:
+    - biotools:{{ name }}
```
Comment (epruesse) [inline recipes/pal2nal/meta.yaml]: You can actually just use the name here, instead of the jinja thing. It's not like that's ever going to change.

**Comment by epruesse** [comment]
@liaochenlanruo This looks good! I send an invitation out for you to join Bioconda, welcome! :)

File: recipes/pal2nal/meta.yaml (hunk 1 above)
Comment (liaochenlanruo) [reply recipes/pal2nal/meta.yaml]: got that, Thanks!

**Comment by liaochenlanruo** [comment]
@bioconda-bot Merge

**Pull request merged**